tests/test_coordinate_validation.py

```
@@ -0,0 +1,725 @@
+        """Document the actual range of CV output coordinates.
+
+        Current state: CV outputs origin-based (0-200, 0-85).
+        Target state: Center-relative (-100 to +100, -42.5 to +42.5).
+        The ETL xy_coordinates.py handles the conversion.
+        """
+        x_min, x_max = cv_csv["x_ice"].min(), cv_csv["x_ice"].max()
+        y_min, y_max = cv_csv["y_ice"].min(), cv_csv["y_ice"].max()
+
+        # Document the actual range (informational — not a pass/fail assertion)
+        # CV pipeline currently outputs origin-based; ETL converts to center-relative
+        is_origin_based = x_max > 100 or y_max > 42.5
```

Ruff (0.15.2)

[error] Local variable `is_origin_based` is assigned to but never used

Remove assignment to unused variable `is_origin_based`

(F841)